packages/nc-gui/assets/style.scss

```diff
@@ -592,6 +605,7 @@ a {
 
 .ant-popover-inner {
   padding: 0 !important;
+  @apply rounded-lg
```

Comment:
merge with L607 also, i.e.

```
@apply rouded-lg !p-0;
```

Reply:
**Fixed in below commit**

[fix : removed unwanted spaces](https://github.com/nocodb/nocodb/pull/7181/commits/4b8ee083f5689fd543f5a727c98a384e9f250525)
[4b8ee08](https://github.com/nocodb/nocodb/pull/7181/commits/4b8ee083f5689fd543f5a727c98a384e9f250525)